doc/getting_started/installation.rst

```diff
@@ -9,7 +9,7 @@ Altair can be installed, along with the example datasets in vega_datasets_, usin
 
 .. code-block:: bash
 
-    pip install altair vega_datasets
+    pip install "altair[all]"
```

Comment:
The quotes are needed for zsh (because square brackets are used for pattern matching), which is the default shell on MacOS. See https://stackoverflow.com/questions/30539798/zsh-no-matches-found-requestssecurity.  I think this approach should be valid across all major shells on windows and linux as well.

Reply:
Ah. I did not know this. Thanks for explaining. Just ignore then!